InterGalacticCurrencyConvertor/src/main/java/IgnConversionException.java

```diff
@@ -0,0 +1,32 @@
+package main.java;
+
+import java.util.Objects;
+
+/**
+ * Stores error information resulting from invalid user input.
+ */
+public final class IgnConversionException extends Exception {
+  private static final long serialVersionUID = 4312898069081147211L;
+  private final IgnConversionErrors mError;
+  private final String mAdditionalInfo;
+
+  public IgnConversionException(IgnConversionErrors error, String additionalInfo) {
+    super();
+    Objects.requireNonNull(error);
+    Objects.requireNonNull(additionalInfo);
+    mError = error;
+    mAdditionalInfo = additionalInfo;
+  }
+
+  public IgnConversionErrors getError() {
+    return mError;
+  }
+
+  public String getErrorMessage() {
```

Comment:
This is not needed. It is easy enough to get from IgnConversionErrors enum method.

Reply:
done